docs/README.md

```diff
@@ -102,6 +102,7 @@ Possible values for the `subject` prop:
 | `specificationProperties`  | List of product specifications currently displayed on the UI. | `{ name: string, value: string }`. Notice: `value` is an optional prop. If omitted, only the specification name (`name`) will be checked. |
 | `areAllVariationsSelected` | Whether all product variations currently available on the UI were selected by the user (`true`) or not (`false`). | No arguments are expected. |
 | `isProductAvailable`                  | Whether the product is available (`true`) or not (`false`).  | No arguments are expected. |
+| `isSellersMoreThan`                  | Whether the quantity of seller is more than argument passed.  | `{ quantity: number }`|
```

Comment:
How about `hasMoreSellersThan`?

Reply:
@lariciamota and @igorbrasileiro what do you think?

Reply:
That sounds better! 🚀

Reply:
realy sounds better, I couldn't think of a better name.